### ISSUE TITLE
Format Handles Whitespace

### ISSUE THREAD

**Issue by entozoon** [panoply/vscode-liquid#36]
Convert things like:
```
{{foo.bar}}         -> {{ foo.bar }}
{%if foo%}          -> {% if foo %}
{{ foo|asset_url }} -> {{ foo | asset_url }} 
```
As it is the recommended syntax, and would greatly improve the crazy inconsistencies in peoples' code 😁

**Comment by panoply** [comment]
This is probably better handled with PrettyDiff/Sparser so I will need to be passed upstream to that project. Somewhat related is  (https://github.com/prettydiff/prettydiff/issues/609).

**Comment by panoply** [comment]
🚢  Shipped [v3.0.0](https://github.com/panoply/vscode-liquid/releases/tag/v3.0.0)
